.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/configuration-reference
+version: 2.1
+
+orbs:
+  docker: circleci/docker@2.0.1
+  github-cli: circleci/github-cli@2.0.0
+
+# Define a job to be invoked later in a workflow.
+# See: https://circleci.com/docs/configuration-reference/#jobs
+jobs:
+
+  build_image_and_deploy:
+    docker:
+      - image: cimg/base:stable
+    # Add steps to the job
+    # See: https://circleci.com/docs/2.0/configuration-reference/#steps
+    steps:
+      - setup_remote_docker:
+          version: 20.10.11
+      - checkout
+      - run:
+          name: "Build docker image and push to Quay"
+          command: |
+            docker login -u="${QUAY_USER}" -p="${QUAY_PASSWORD}" quay.io
+            docker build -t quay.io/astronomer/astro-trial-examples .
+            docker push quay.io/astronomer/astro-trial-examples
+
+
+
+# Invoke jobs with workflows
+# See: https://circleci.com/docs/2.0/configuration-reference/#workflows
+workflows:
+  version: 2.1
+  build-and-deploy:
+    jobs:
+      - build_image_and_deploy:
+          context:
+             - astro-example-dags
+          # filters:
+          #   branches:
+          #     only:
+          #       - main
```

Comment:
should we uncomment this? or at least tag images built on branches with something other than `latest`?